packetbeat/protos/graphite/config.go

```diff
@@ -0,0 +1,25 @@
+// Author: Sudhindra Sajjal <sudhindra.s@media.net> & Sumit Sulakhe <sumit.su@media.net>
```

Comment:
There should not be any copyright or notes at the top of the files as this is covered by the LICENSES file and the CLA.